ambassador/ambassador/config.py

```diff
@@ -1535,6 +1546,44 @@ def module_config_ratelimit(self, ratelimit_config):
 
         return (filter, grpc_service)
 
+    def module_config_tracing(self, tracing_config):
+        cluster_hosts = None
+        driver = None
+        driver_config = None
+        host_rewrite = None
+        sources = []
+
+        if tracing_config:
+            for config in tracing_config.values():
+                sources.append(config['_source'])
+                cluster_hosts = config.get("service", None)
+                driver = config.get("driver", None)
+                driver_config = config.get("config", {})
+                host_rewrite = config.get("host_rewrite", None)
```

Comment:
looking back at this, should `host_rewrite` be a field in TracingService? And if so, should it be documented?

Reply:
If I'm not mistaken it's also an undocumented field of AuthService and RateLimitService. I'm not absolutely sure what was the intent in the original AuthService, but I agree, it should either be documented or removed.